packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.test.tsx

```diff
@@ -556,15 +561,20 @@ describe('DetailsHeader', () => {
     expect(header._currentDropHintIndex).toBe(5);
     dropHintElement = component.find('#columnDropHint_5').getDOMNode();
     dropHintElementChildren = dropHintElement.children;
-    expect(dropHintElementChildren.item(0).getAttribute('style')).toEqual('visibility: visible;');
+    console.log(dropHintElementChildren.item(0).getAttribute('style'));
```

Comment:
Please remove console.log statements before merging.

Reply:
Fixed in https://github.com/OfficeDev/office-ui-fabric-react/pull/7346/commits/59e41426da6ad8ab3370208a35fc062e4027e161, nice catch 👀.